resources/views/admin/orders/list.blade.php

```diff
@@ -99,7 +91,7 @@
 			'21st', '22nd', '23rd', '24th', '25th',
 			'26th', '27th', '28th', '29th', '30th',
 			'31st'
-		],
+		].slice(0, {{ count($monthly_invoices) }}),
```

Comment:
This just chops off any extra days from the graph so in a 28 day month, you don't see up to 31.